core/google/cloud/_helpers.py

```diff
@@ -168,134 +159,13 @@ def _ensure_tuple_or_list(arg_name, tuple_or_list):
-              config file.
-    """
-    appdata_dir = os.getenv('APPDATA', '')
-    return os.path.join(appdata_dir, _GCLOUD_CONFIG_FILE)
-
-
-def _default_service_project_id():
-    """Retrieves the project ID from the gcloud command line tool.
-
-    This assumes the ``.config`` directory is stored
-    - in ~/.config on *nix systems
-    - in the %APPDATA% directory on Windows systems
-
-    Additionally, the ${HOME} / "~" directory may not be present on Google
-    App Engine, so this may be conditionally ignored.
-
-    Files that cannot be opened with configparser are silently ignored; this is
-    designed so that you can specify a list of potential configuration file
-    locations.
-
-    :rtype: str or ``NoneType``
-    :returns: Project-ID from default configuration file else ``None``
-    """
-    search_paths = []
-    if _USER_ROOT is not None:
-        search_paths.append(_get_nix_config_path())
-
-    if os.name == 'nt':
-        search_paths.append(_get_windows_config_path())
-
-    config = configparser.RawConfigParser()
-    config.read(search_paths)
-
-    if config.has_section(_GCLOUD_CONFIG_SECTION):
-        try:
-            return config.get(_GCLOUD_CONFIG_SECTION, _GCLOUD_CONFIG_KEY)
-        except configparser.NoOptionError:
-            return None
-
-
-def _compute_engine_id():
-    """Gets the Compute Engine project ID if it can be inferred.
-
-    Uses 169.254.169.254 for the metadata server to avoid request
-    latency from DNS lookup.
-
-    See https://cloud.google.com/compute/docs/metadata#metadataserver
-    for information about this IP address. (This IP is also used for
-    Amazon EC2 instances, so the metadata flavor is crucial.)
-
-    See https://github.com/google/oauth2client/issues/93 for context about
-    DNS latency.
-
-    :rtype: str or ``NoneType``
-    :returns: Compute Engine project ID if the metadata service is available,
-              else ``None``.
-    """
-    host = '169.254.169.254'
-    uri_path = '/computeMetadata/v1/project/project-id'
-    headers = {'Metadata-Flavor': 'Google'}
-    connection = http_client.HTTPConnection(host, timeout=0.1)
-
-    try:
-        connection.request('GET', uri_path, headers=headers)
-        response = connection.getresponse()
-        if response.status == 200:
-            return response.read()
-    except socket.error:  # socket.timeout or socket.error(64, 'Host is down')
-        pass
-    finally:
-        connection.close()
-
-
-def _get_production_project():
-    """Gets the production project if it can be inferred."""
-    return os.getenv(PROJECT)
-
-
 def _determine_default_project(project=None):
     """Determine default project ID explicitly or implicitly as fall-back.
 
     In implicit case, supports three environments. In order of precedence, the
     implicit environments are:
 
-    * GOOGLE_CLOUD_PROJECT environment variable
+    * GOOGLE_CLOUD_PROJECT and GCLOUD_PROJECT environment variable
```

Comment:
1. "environment variable" --> "environment variables"
1. Shouldn't this section just be scrapped and point to the docs for `google.auth.default`?